examples/laravel.yml

```diff
@@ -19,17 +19,20 @@ services:
     links:
       - eatapp_redis:redis
     labels:
-      - "traefik.http.routers.web.rule=Host(`eatapp.srvz-webapp.he-arc.ch`)"
+      - "traefik.enable=true"
+      - "traefik.http.services.eatapp.loadbalancer.server.port=80"
+      - "traefik.http.routers.eatapp.rule=Host(`eatapp.srvz-webapp.he-arc.ch`)"
+      - "traefik.http.routers.eatapp.entrypoints=web-secure"
+      - "traefik.http.routers.eatapp.tls.certresolver=letsencrypt"
 
   eatapp_redis:
     image: redis:5-alpine
     volumes:
       - eatapp_redis:/data
 
 networks:
-  default:
-    external:
-      name: webapp-net
+  web:
+    external: true
```

Comment:
_Explicit is better than implicit_ mais ça demande pas mal de changements.

Reply:
Yes, it is better for security purpose and readability